clients/admin-ui/package.json

```diff
@@ -2,25 +2,25 @@
   "name": "admin-ui",
   "private": true,
   "scripts": {
-    "dev": "next dev",
-    "dev:mock": "echo '🚨 Running with mock API'; NEXT_PUBLIC_MOCK_API=true next dev",
-    "build": "next build",
-    "start": "next start",
-    "lint": "eslint . --ext .ts,.tsx",
-    "lint:fix": "eslint . --fix --ext .ts,.tsx",
-    "format": "prettier --write src __tests__/",
-    "format:ci": "prettier --check src __tests__/",
-    "test": "jest --watch",
-    "test:ci": "jest",
     "analyze": "cross-env ANALYZE=true next build",
-    "analyze:server": "cross-env BUNDLE_ANALYZE=server next build",
     "analyze:browser": "cross-env BUNDLE_ANALYZE=browser next build",
-    "export": "next build && next export",
+    "analyze:server": "cross-env BUNDLE_ANALYZE=server next build",
+    "build": "next build",
     "chrome:debug": "BROWSER='google chrome' BROWSER_ARGS='--remote-debugging-port=9222' npm run dev",
     "cy:open": "cypress open",
     "cy:run": "cypress run",
     "cy:start": "next build && NODE_ENV=test next start",
-    "openapi:generate": "openapi --input http://localhost:8080/openapi.json --output ./src/types/api --exportCore false --exportServices false --indent 2"
+    "dev": "next dev",
+    "dev:mock": "echo '🚨 Running with mock API'; NEXT_PUBLIC_MOCK_API=true next dev",
+    "export": "next build && next export",
+    "format": "prettier --write src __tests__/",
+    "format:ci": "prettier --check src __tests__/",
+    "lint": "eslint . --ext .ts,.tsx",
+    "lint:fix": "eslint . --fix --ext .ts,.tsx",
+    "openapi:generate": "openapi --input http://localhost:8080/openapi.json --output ./src/types/api --exportCore false --exportServices false --indent 2",
+    "start": "next start",
+    "test": "jest --watch",
+    "test:ci": "jest"
```

Comment:
This will probably cause merge conflicts 🙊

Reply:
There is already a discrepancy between FidesOps, FidesCtl, and Privacy Center given their respective package.json files. Your right in that this might need to me merged manually via the merge conflict.